zion/src/main/scala/edu/uci/ics/cloudberry/zion/actor/DataStoreManager.scala

```diff
@@ -220,14 +220,13 @@ class DataStoreManager(metaDataset: String,
     val actor = context.child("data-" + query.dataset).getOrElse {
       val info = metaData(query.dataset)
       val schema: Schema = info.schema
-      val initCardinality: Cardinality = new Cardinality(info.dataInterval.getStart, info.dataInterval.getEnd, info.stats.cardinality)
       info.createQueryOpt match {
         case Some(_) =>
-          val ret = childMaker(AgentType.View, context, "data-" + query.dataset, query.dataset, schema, initCardinality, queryGenFactory(), conn, config)
+          val ret = childMaker(AgentType.View, context, "data-" + query.dataset, query.dataset, schema, None, queryGenFactory(), conn, config)
```

Comment:
because the `viewAgent` doesn't need the `dataSetInfo` now. You can see the childMaker function.